### ISSUE TITLE
Changing "frames_tres" causes R to hang and zero output

### ISSUE THREAD

**Issue by wlphipps** [16EAGLE/moveVis#18]
Hello Jakob,
Thanks for creating this fantastic package and the accompanying guidance.

I have generated several successful .mov outputs using my own data, but when I define "frames_tres = 60" [i.e. asking for a 60 second .mov output] this causes R to "hang" and no output is ever generated and the run does not seem to complete. Could this be a computer performance issue during the paths interpolation process or am I missing something obvious? 

Here is the code (which works fine when frames_tres is not specified, but takes several hours to complete). The dataset consists of 12,133 observations, requiring generation of 6,292 frames:

animate_move(data_ani, out_dir, conv_dir = conv_dir, tail_elements = 10,
             paths_mode = "true_data", frames_nmax = 0, frames_fps = 100,
             img_caption = img_caption, img_title = img_title,
             img_sub = img_sub, log_level = 1, scalebar_dist = 1000, frames_tres = 60, out_format = "mov", out_name = "Rupis20172018")
## Result:##
Checking prerequisites...
Detected 'conv_dir' executable on this system: 'ffmpeg'
Paths interpolation will be applied, since 'frames_tres' is user-defined.
Computing uniform frame times from timestamps...
> 
## end ##

Thank you in advance,
Louis

**Comment by 16EAGLE** [comment]
Hi Louis, sorry for the late reply. ``frames_tres`` defines the uniform temporal resolution of your data, not the overall duration of the output. It triggers a linear interpolation in case it is defined or that the time difference of your timestamps is irregular. To avoid interpolation to a regular time sampling, I recommend to prepare your data before calling ``animate_move()`` to the wished temporal (regular) resolution (e.g. if you want a 60 seconds animation, considering 25 frames per seconds, you need 60*25 timestamps representing your movement as input, if you want to manually prepare your data).

I don't know why, in your case, the interpolation does take that long, ``animate_move`` performs a simple call of ``interpolateTime`` of the ``move`` package. Could you try to manually interpolate your data beforehand with that call and than hand it to ``animate_move``?

Hope I understood you correctly.

**Comment by wlphipps** [comment]
Hi Jakob,
Thanks for this explanation - exactly as I thought, I had misinterpreted the function of frames_tres. Two of the animals had much finer temporal resolution data compared to the majority which might have contributed to the processing time (?). When I come back to this analysis I will manipulate the data so that they have the same temporal resolution for each individual. To get an animation of a certain length (e.g. ~60 seconds) I changed the value of frames_nres depending on the total number of frames, which worked fine. 
I will update regarding frames_tres at a later date.
Thanks again,
Louis
